pkgs/development/libraries/libsoup/3.x.nix

```diff
@@ -0,0 +1,102 @@
+{ stdenv
+, lib
+, fetchurl
+, glib
+, meson
+, ninja
+, pkg-config
+, gnome
+, libsysprof-capture
+, sqlite
+, glib-networking
+, gobject-introspection
+, withIntrospection ? stdenv.buildPlatform == stdenv.hostPlatform
+, vala
+, withVala ? stdenv.buildPlatform == stdenv.hostPlatform
```

Comment:
Shouldn't this be one input called crossCompile or similar?

Reply:
Previously, `stdenv.isCross` was used but the cross people switched to this.